src/backend/marsha/settings.py

```diff
@@ -397,7 +397,7 @@ class Production(Base):
 
     # folder where static will be stored. It matches the path_pattern used
     # in the cloudfront configuration.
-    AWS_LOCATION = Base.STATIC_URL
+    AWS_LOCATION = Base.STATIC_URL[1:]
```

Comment:
did you try removing the / from STATIC_URL too?

Reply:
Unfortunately it doesn't work.

Reply:
Then using `strip` may be safer and more explicit:
```
STATIC_URL.lstrip('/')
```

Reply:
:+1: